src/CSFML/Graphics/ConvertTransform.hpp

```diff
@@ -47,7 +48,5 @@
 [[nodiscard]] inline sf::Transform convertTransform(const sfTransform& transform)
 {
     const float* m = transform.matrix;
-    return {m[0], m[1], m[2],
-            m[3], m[4], m[5],
-            m[6], m[7], m[8]};
+    return {m[0], m[1], m[2], m[3], m[4], m[5], m[6], m[7], m[8]};
```

Comment:
Can we keep this as a matrix?

Reply:
Sure I can add some `// clang-format-off` comments like SFML has.